paper-server/src/main/java/com/destroystokyo/paper/PaperVersionFetcher.java

```diff
@@ -84,33 +84,32 @@ public static void getUpdateStatusStartupMessage(final String repo, final Server
             switch (distance) {
                 case DISTANCE_ERROR -> COMPONENT_LOGGER.error(text("*** Error obtaining version information! Cannot fetch version info ***"));
                 case 0 -> {
-                    if (!newVersionAvailable) {
-                        COMPONENT_LOGGER.info(text("*** You are running the latest version! ***", NamedTextColor.GREEN));
-                    } else {
+                    if (newVersionAvailable) {
                         COMPONENT_LOGGER.info(text("*************************************************************************************", NamedTextColor.GREEN));
                         COMPONENT_LOGGER.info(text("You are running the latest build for your Minecraft version (" + build.minecraftVersionName() + ")", NamedTextColor.GREEN));
                         COMPONENT_LOGGER.info(text("However, there is a new Minecraft version available on the downloads page (" + newVersion + ")!", NamedTextColor.GREEN));
-                        COMPONENT_LOGGER.info(text("It is recommended that you download it as soon as possible (unless it is experimental)", NamedTextColor.GREEN));
```

Comment:
You can add the (unless it is experimental) back unless you genuinely plan to check if the version is experimental or not